Runtime/ApiRequests.cs

```diff
@@ -11,9 +12,15 @@ internal class ApiRequests
     {
         private static readonly string ApiEndpoint = "https://backend.blockadelabs.com/api/v1/";
 
-        public static async Task<T> GetAsync<T>(string path, string apiKey)
+        public static async Task<T> GetAsync<T>(string path, string apiKey, params (string, string)[] queryParams)
         {
-            using var request = UnityWebRequest.Get(ApiEndpoint + path + "?api_key=" + apiKey);
+            var queryString = "?api_key=" + apiKey;
+            if (queryParams.Length > 0)
+            {
+                queryString += "&" + string.Join("&", queryParams.Select(kv => kv.Item1 + "=" + kv.Item2));
```

Comment:
We may want to escape the parameter keys and values.

```suggestion
                queryString += "&" + string.Join("&", queryParams.Select(kv => UnityWebRequest.EscapeURL(kv.Item1) + "=" + UnityWebRequest.EscapeURL(kv.Item2)));
```